e3sm_to_cmip/cmor_handlers/handler.py

```diff
@@ -257,7 +256,18 @@ def cmorize(
                 positive=self.positive,
             )
 
-            self._cmor_write(ds, cmor_var_id, cmor_ips_id, time_dim, time_bnds_key)
+            if time_dim is None:
+                self._cmor_write(ds, cmor_var_id)
+            else:
+                self._cmor_write_with_time(ds, cmor_var_id, time_dim, cmor_ips_id)
+
+        # NOTE: It is important to close the CMOR module AFTER CMORizing all of
+        # the variables. Otherwise, the IDs of cmor objects gets wiped after
+        # every loop.
```

Comment:
An extremely important note about `cmor` that I just learned from working on this issue.